common/src/main/res/values/styles.xml

```diff
@@ -178,10 +178,6 @@
         <item name="android:textSize">14sp</item>
     </style>
 
-    <style name="MontserratRegular.MG.15">
```

Comment:
Not needed anymore. Some of the screens I did (like "see passphrase" screen) can use the new styles, but there are small inconsistencies with the design system because Ros did them earlier.